### DISCUSSION TITLE
Windows aks pools

### DISCUSSION THREAD

**Discussion by DienasV** [kubernetes-sigs/blob-csi-driver#1655] [Q&A] [answered]
Hi guys in the doc it says that only fuse method doesnt work on windows pools or im wrong ?

**Comment by andyzhangx** [comment, marked as answer]
this blob csi driver does not support Windows node.

**Reply by ayushdg07** [reply]
Hi @andyzhangx 
Is this still not supported? Are there any plans to support it in the future?

**Reply by andyzhangx** [reply]
No, the blobfuse driver does not support Windows.
